snippets/rust.snippets

```diff
@@ -19,32 +19,53 @@ snippet bench "Bench function" b
 			${0}
 		})
 	}
-snippet new
+snippet new "Constructor function"
 	pub fn new(${2}) -> ${1:Name} {
-		${0}return $1 { ${3} };
+		$1 { ${3} };
 	}
-snippet main
+snippet main "Main function"
 	pub fn main() {
 		${0}
 	}
-snippet let
-	let ${1:name: type} = ${2};
-snippet let
-	let mut ${1:name: type} = ${2};
-snippet pln
+snippet let "let variable declaration"
+	let ${1:name}${2:: ${3:type}} = ${4};
+snippet letm "let mut variable declaration"
+	let mut ${1:name}${2:: ${3:type}} = ${4};
```

Comment:
not supporting them was intentional, because having them broke some snipMate snippets. that they now support them suckz. Do they now support it for all snippets or only for version 2?